packages/common/src/content.ts

```diff
@@ -885,6 +891,66 @@ const getSolutionUrl = (content: IHubContent): string => {
   return hubUrl;
 };
 
+/**
+ * Create a new content with updated boundary properties
+ * @param content original content
+ * @param boundary boundary provenance
+ * @returns
+ */
+export const setContentBoundary = (
```

Comment:
Is this being used anywhere aside from the test?

Reply:
not yet, but it will be used by opendata-ui